aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -9,79 +9,108 @@
 ###########################################################################
 # pylint: disable=fixme,too-many-branches,too-many-locals,too-many-statements,too-many-arguments
 """Provides export functionalities."""
+from collections import defaultdict
 import logging
 import os
 import tarfile
 import time
+from typing import (
+    Any,
+    Callable,
+    DefaultDict,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Set,
+    Tuple,
+    Union,
+    cast,
+)
 
 from aiida import get_version, orm
 from aiida.common import json
 from aiida.common.exceptions import LicensingException
-from aiida.common.folders import RepositoryFolder, SandboxFolder, Folder
+from aiida.common.folders import Folder, RepositoryFolder, SandboxFolder
 from aiida.common.lang import type_check
-from aiida.common.log import override_log_formatter, LOG_LEVEL_REPORT
+from aiida.common.log import LOG_LEVEL_REPORT, override_log_formatter
```

Comment:
hehe, wow - things are even ordered alphabetically now?
I wonder at which points the import lines also have to be in alphabetic order ;-)

Reply:
yep I run it with this in the pre-commit 😄 

```yaml
- repo: https://github.com/timothycrosley/isort
  rev: 5.5.3
  hooks:
  - id: isort
```

Reply:
does it all for you!